tests/feature_tests/pivoting/Makefile

```diff
@@ -2,7 +2,7 @@ M3PATH := "$(dirname "$(readlink -f "${0}")")../../../"
 export CONTROL_PLANE_MACHINE_COUNT := 3
 export WORKER_MACHINE_COUNT := 1
 
-all: provision fetch_manifests pivoting
+all: provision fetch_manifests pivoting fetch_target_logs
```

Comment:
This addition of fetch_target_logs will cover the log collection for feature tests.